packages/sdk-aws-kms-signer/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "@dfns/sdk-aws-kms-signer",
```

Comment:
nit, `sdk-awskmssigner` to have fewer dashes? otherwise looks good, test and :shipit:

Reply:
Need to figure out our dash acceptance criteria 😂 , I've udpated

Reply:
😆